sdk/tests/utilities/test_api_client_factory_auth_flow.py

```diff
@@ -0,0 +1,63 @@
+import unittest
+from lusid import InstrumentsApi, ResourceListOfInstrumentIdTypeDescriptor
+from lusid.utilities import ApiClientFactory
+from utilities import CredentialsSource
+from unittest.mock import patch
+
+source_config_details, config_keys = CredentialsSource.fetch_credentials(), CredentialsSource.fetch_config_keys()
+
+
+class ApiFactory(unittest.TestCase):
```

Comment:
for all these tests we need to explicitly clear and set the env vars,  see https://github.com/finbourne/lusid-sdk-python-preview/blob/master/sdk/tests/utilities/test_api_client_builder.py#L211 for an example on how to do this

Reply:
Sure, that is now completed.